src/plugins/intel_gpu/tests/test_cases/reorder_gpu_test.cpp

```diff
@@ -2467,6 +2467,8 @@ class ReorderTest : public ::testing::TestWithParam<T> {
     std::vector<primitive_id> executed_prims;
 
     void execute(T& p) {
+        if (!engine.get_device_info().supports_immad)
+            return;
```

Comment:
I think this case is for both onednn and cldnn..

Reply:
Thanks. Reverted that change.